src/content/docs/en/guides/markdown-content.mdx

```diff
@@ -502,3 +378,24 @@ const {frontmatter} = Astro.props;
 You can also [style your Markdown](/en/guides/styling/#markdown-styling) in your layout component.
 
 <ReadMore>Learn more about [Markdown Layouts](/en/basics/layouts/#markdown-layouts).</ReadMore>
+
+## Fetching Remote Markdown
+
+**Astro does not include built-in support for remote Markdown outside of [experimental content collections](/en/reference/configuration-reference/#experimentalcontentlayer)!**
```

Comment:
```suggestion
**Astro does not include built-in support for remote Markdown outside of [experimental content layer](/en/reference/configuration-reference/#experimentalcontentlayer)!**
```

Reply:
This is actually fine as is! The feature is still content collections, just the underlying API that powers them is experimental. The v5 docs will say "collections" just without the word experimental!